webapp/src/components/containers/ButtonsCard.vue

```diff
@@ -39,7 +43,7 @@ import CustomButton from "@/components/simple/CustomButton.vue";
 const props = withDefaults(
   defineProps<{
     titleAlign?: "left" | "center" | "right";
-    buttons: List<readonly [string, () => void]>;
+    buttons: List<readonly [string, () => void, string?]>;
```

Comment:
```suggestion
    buttons: List<readonly [string, () => void, string?]>; // button text, on click function, optional description
```
Comment to clarify the different fields